test/Bech32m.t.sol

```diff
@@ -1,5 +1,5 @@
 // Solidity Version: ^0.8.13
```

Comment:
nit: add license header instead of this

Reply:
Yes. Great idea. I guess it should be added to all files. So maybe create a new PR with a license.